products/HiveMQ

```diff
@@ -0,0 +1,93 @@
+---
+title: HiveMQ
+category: server-app
+permalink: /HiveMQ
+alternate_urls:
+-   /Hive-MQ
+releasePolicyLink: "https://www.hivemq.com/blog/hivemq-moves-to-new-release-cadence/"
+releaseColumn: true
+releaseDateColumn: true
+
+releases:
+-   releaseCycle: "4.13"
```

Comment:
If I understood correctly this is the rolling release. So I think the `releaseCycle` should be `4` (a `releaseLabel` could be added to make it more obvious this is a rolling release), and we'll use `latest` to track the monthly releases.